flink-runtime/src/main/java/org/apache/flink/runtime/minicluster/MiniCluster.java

```diff
@@ -120,12 +125,20 @@
 import java.util.function.Function;
 import java.util.stream.Collectors;
 
+import static org.apache.flink.configuration.ConfigOptions.key;
 import static org.apache.flink.util.Preconditions.checkNotNull;
 import static org.apache.flink.util.Preconditions.checkState;
 
 /** MiniCluster to execute Flink jobs locally. */
 public class MiniCluster implements AutoCloseableAsync {
 
+    public static final ConfigOption<String> PLUGIN_DIRECTORY =
```

Comment:
Do we need a config option here? I was thinking that it would be better in this case to add a simple `String` field to `MiniClusterConfiguration` iff the option is only ever used in the `MiniCluster`. Then we can make it easier to use in `MiniClusterConfiguration#Builder`.

Reply:
I think this depends a bit if a user only wants to execute a jar locally. In this case one usually does not interact with the minicluster it is only started in the background by the `LocalExecutionEnvironment`. 

Perhaps one option can be to offer both possibilities but this may also confuse users. WDYT?

Reply:
If you want to support `LocalExecutionEnvironment` then, we should move the options to somewhere more visible. Then the question arises why we are not supporting all envs ;).

Or we just say that for `MiniCluster` that we just use `MiniClusterConfiguration` and for everything else we use the `ENV_FLINK_PLUGINS_DIR`?